float32 max_angle

---
bool result
